db/python/utils.py

```diff
@@ -273,7 +273,10 @@ def __post_init__(self):
                 setattr(self, field.name, GenericFilter(eq=value))
 
     def to_sql(
-        self, field_overrides: dict[str, str] = None
+        self,
+        field_overrides: dict[str, str] = None,
+        only: list[str] | None = None,
+        exclude: list[str] | None = None,
```

Comment:
Nitpick: could we make include / exclude a set, make the `in` a constant operation, not that I think it really changes much, but good that we're acknowledging the type of operation we're planning to run.